libs/ledger-live-common/src/families/stellar/api/horizon.ts

```diff
@@ -107,24 +121,25 @@ export const fetchAccount = async (
   spendableBalance: BigNumber;
   assets: BalanceAsset[];
 }> => {
-  let account: typeof AccountRecord = {};
-  let balance: Record<string, any> = {};
+  let account: ServerApi.AccountRecord = {} as ServerApi.AccountRecord;
   let assets: BalanceAsset[] = [];
+  let balance = "0";
 
   try {
     account = await server.accounts().accountId(addr).call();
-    balance = account.balances?.find(balance => {
-      return balance.asset_type === "native";
-    });
+    balance =
+      account.balances.find(balance => {
```

Comment:
According to current code there's no guarantee that `account.balances` isn't undefined, maybe better to keep the safety? (even if types pretend it's non-optional, backend APIs don't always respect this, and types don't protect us against runtime data...)

(also below L135)

Reply:
Okay I think that this lib is not the best maintained indeed if the types can't be trusted